gcloud-java-bigquery/src/main/java/com/google/cloud/bigquery/TableInfo.java

```diff
@@ -63,6 +64,7 @@ public Table apply(TableInfo tableInfo) {
   private final Long creationTime;
   private final Long expirationTime;
   private final Long lastModifiedTime;
+  private final TimePartitioning timePartitioning;
```

Comment:
Thanks for the review! I would definitely appreciate it if you took care of these changes yourself.